packages/auth/src/AuthCard/AuthCard.tsx

```diff
@@ -33,38 +33,30 @@ export const AuthCard = ({
   const { logo } = React.useContext(LogoContext);
 
   return (
-    <Card variant="forms.auth.cardContainer">
+    <Card variant="primary" sx={{ maxWidth: '340px' }}>
       <Flex sx={{ flexDirection: 'column', gap: 3 }}>
         {logo && (
           <Flex sx={{ width: '100%', justifyContent: 'center' }}>{logo}</Flex>
         )}
 
         {children}
 
-        <Flex variant="forms.auth.submitButtonContainer">
-          <Button
-            type="submit"
-            aria-label="submit-login"
-            variant="forms.auth.submitButton"
-          >
+        <Flex sx={{ justifyContent: 'space-between', marginTop: 7 }}>
+          <Button type="submit" aria-label="submit-login" variant="primary">
             {buttonLabel}
           </Button>
           {forgotPassword && (
-            <Link variant="forms.auth.link" onClick={forgotPassword.onClick}>
+            <Link variant="primary" onClick={forgotPassword.onClick}>
               {forgotPassword.label}
             </Link>
           )}
         </Flex>
 
-        <Flex variant="forms.auth.containerLinks">
+        <Flex variant="forms.containerLinks">
```

Comment:
I don't think `containerLinks` make sense here because it isn't probably that we need this variant in other forms. Here we can use `sx` instead of `variant`.

In the future, it'd be interesting to create [container](https://theme-ui.com/components/container) groups and use them, but we need to do this with a designer.